app/locales/zh/translations.js

```diff
@@ -56,6 +56,7 @@ export default {
       home: "主頁",
       browse: "瀏覽",
       myorders: "我的訂單",
+      myDashboard: "My Dashboard",
```

Comment:
I think above translation is not in this file?

Reply:
I think i need to check the translation file. I'll check and update. Sorry for the trouble guys.